src/config.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Gregorio configuration headers. Copyright (C) 2015 Elie Roux
```

Comment:
Legally speaking, The Gregorio Project doesn't mean anything, while authors are listed in CONTRIBUTORS.md... but I'm not sure it matters, nor that any lawyer could say that there is a strict difference...